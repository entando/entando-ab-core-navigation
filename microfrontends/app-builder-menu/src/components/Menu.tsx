export function Menu() {
    return (
        <menu>
            This is the menu
        </menu>
    )
}
