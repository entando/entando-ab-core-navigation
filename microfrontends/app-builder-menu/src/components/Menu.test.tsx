import { render } from "@testing-library/react";
import { Menu } from "./Menu";

describe("Menu Test", () => {
    it("must be rendered", () => {
        const { getByText } = render(<Menu/>)

        expect(getByText("This is the menu")).toBeInTheDocument()
    })

    it("should match snapshot", () => {
        const { container } = render(<Menu/>)

        expect(container).toMatchSnapshot()
    })
})
