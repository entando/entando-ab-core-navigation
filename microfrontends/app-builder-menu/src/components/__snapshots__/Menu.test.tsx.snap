// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Test should match snapshot 1`] = `
<div>
  <menu>
    This is the menu
  </menu>
</div>
`;
